libraries/pushstore/impl/src/main/kotlin/io/element/android/libraries/pushstore/impl/UserPushStoreDataStore.kt

```diff
@@ -80,5 +100,7 @@ class UserPushStoreDataStore(
         context.dataStore.edit {
             it.clear()
         }
+        // Also delete the file
+        context.preferencesDataStoreFile(preferenceName).delete()
```

Comment:
Now that the preference are emptied, the file can be removed as well to keep the local storage tidied up.